src/Storefront/Theme/ThemeCompiler.php

```diff
@@ -129,9 +129,9 @@ public function compileTheme(
         }
     }
 
-    public static function getThemePrefix(string $salesChannelId, string $themeId): string
+    public static function getThemePrefix(string $salesChannelId, string $themeName): string
```

Comment:
Keeping it as `themeId` or maybe `themeIdentifier`to differentiate it from the field, makes more sense, if it's configurable.